### DISCUSSION TITLE
Where in the API I can find where to remove this string ?

### DISCUSSION THREAD

**Discussion by Fioranit** [tulir/whatsmeow#530] [General]
![Screenshot_2024-01-15-09-08-07-28_6012fa4d4ddec268fc5c7112cbb265e7](https://github.com/tulir/whatsmeow/assets/130459111/d4681e7e-e1a4-48ac-a6b0-72e7d7b90cc3)

I would like to change "whatsmeow" in this screen. Please, I would like to learn where in the thousands of locations of the whatsmeow API I need to change to eliminate this string. This screenshot is from WhatsApp business for Android, my iOS WhatsApp the it does not show. 
Thanks I'm advance!

**Comment by arugaz** [comment]
#469
